src/main/java/games/strategy/engine/ClientFileSystemHelper.java

```diff
@@ -32,55 +30,46 @@ private ClientFileSystemHelper() {}
   /**
    * @return Folder that is the 'root' of the tripleA binary installation. This folder and
    *         contents contains the versioned content downloaded and initially installed. This is
-   *         in contrast to the user root folder that is not replaced between between installations.
+   *         in contrast to the user root folder that is not replaced between installations.
    */
   public static File getRootFolder() {
-    final String fileName = getGameRunnerFileLocation(GameRunner.class.getSimpleName() + ".class");
+    final String classFilePath = getGameRunnerClassFilePath();
```

Comment:
The single parameter to this method was inlined because the argument was essentially a constant.